cli/generators/component/templates/component.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
```

Comment:
If most of them will actually use them, I think it's fine. It's going to save you more time if it has it and you (sometimes) need to remove it than if you don't have it and almost every time you need to add it.